controller/src/main/java/io/pravega/controller/timeout/TimerWheelTimeoutService.java

```diff
@@ -95,8 +94,8 @@ public void run(Timeout timeout) throws Exception {
                         // at a later point of time.
                         if (ex != null) {
                             Throwable error = getRealCause(ex);
-                            if (error instanceof WriteConflictException ||
-                                    error instanceof DataNotFoundException ||
+                            if (error instanceof StoreException.WriteConflictException ||
+                                    error instanceof StoreException.NodeNotFoundException ||
                                     error instanceof TransactionNotFoundException ||
```

Comment:
`TransactionNotFoundException` could be mapped to `StoreException.NodeNotFound`, and `OperationOnTxNotAllowedException` could be mapped to `StoreException.IllegalState` isn't it?